cmp/viewmanager/View.ts

```diff
@@ -33,24 +47,25 @@ export class View<T extends PlainObject = PlainObject> {
         return this.info?.lastUpdated ?? null;
     }
 
-    get token(): string {
-        return this.info?.token ?? null;
+    get typedName(): string {
```

Comment:
Wonder if it would make sense to have a getTypedName helper that takes a view and a model or view and a type rather than having it be a top-level property on view itself